### ISSUE TITLE
Extend "no unlikely changes" functionality to work with likely changes as well

### ISSUE THREAD

**Issue by samm82** [balacij/Drasil#57]
Relevant code:
https://github.com/balacij/Drasil/blob/e5af4bfdf741b6e0c9d19eaa0f8ddae913b4093d/code/drasil-docLang/lib/Drasil/DocumentLanguage.hs#L403-L420

**Comment by balacij** [comment]
Good idea! I'll make sure to do this and send it upstream before closing this ticket.
